config/rbac/role.yaml

```diff
@@ -18,6 +18,7 @@ rules:
   resources:
   - services
   - configmaps
+  - secrets
```

Comment:
Why does RMO now need to create/delete secrets? From the code below it looks like RMO just needs to GET secrets.

Reply:
+1, seperated this out with get, list and watch permissions only for resource secrets.